### DISCUSSION TITLE
Is there a way to make Aural to remember which song (from a playlist) was last played (or stopped at)?

### DISCUSSION THREAD

**Discussion by barseghyanartur** [kartik-venugopal/aural-player#80] [Q&A] [answered]
Is there a way to make Aural to remember which song (from a playlist) was last played (or stopped at)?

**Comment by kartik-venugopal** [comment]
Hi @barseghyanartur , 

I assume you mean a "playlist" as in an M3U playlist file that you import into Aural ?

No, currently, there is no way to make Aural remember the last played song from a specific playlist file. Once a playlist file is imported into Aural, all the historical info is in terms of the individual tracks only ... the association with the playlist file is lost.

There is history for all tracks played by the app, including a feature to "resume last played track".

That said, if you import a playlist file and there are no other tracks in the Aural playlist, then you stop playback, the app history will effectively remember the last played track and position from the playlist, but this is a special case of course.

Does this help ?

**Reply by barseghyanartur** [reply]
No, I mean I just loaded a directory into Aural and started to play. It played until song 5. Then I restart. Start Aural. I see the songs loaded into Aural, but I don't see which one was the last song played.

**Reply by kartik-venugopal** [reply, marked as answer]
Ah ok, then both those things are easy to do in Aural :)

1 - To find the last song played, go to History > Recently Played in the menu bar. The last played song will be at the top of the list. The items in that list are sorted chronologically with the most recent items first. And they are also grouped into categories like "Past 24 hours", "Past 7 days", etc. See image below:

![recentlyPlayed](https://github.com/kartik-venugopal/aural-player/assets/30028539/cb82b395-e1ca-48f6-b80e-0e2a3e701cce)

2 - To resume the last played song from the position at which it was stopped, go again to the History menu, and there is a menu item there titled "Resume last played track". So you can easily resume playback from wherever you stopped before. See image below:

![resume](https://github.com/kartik-venugopal/aural-player/assets/30028539/5847a768-9640-4b82-af75-f65c64c778de)
